From aa5d3b25b64cbbfb8240056a0d8d3f72df385fa9 Mon Sep 17 00:00:00 2001
From: Yong Fu <fuyong@neusoft.com>
Date: Thu, 16 Jul 2020 17:43:30 +0800
Subject: [PATCH 1/3] Update-package-versioning-for-TIS-format

Update the package version and set the docker version and commit information

Signed-off-by: Yong Fu <fuyong@neusoft.com>
---
 SPECS/docker-ce-cli.spec | 6 ++++--
 1 file changed, 4 insertions(+), 2 deletions(-)

diff --git a/SPECS/docker-ce-cli.spec b/SPECS/docker-ce-cli.spec
index 25fae60..5b496d4 100644
--- a/SPECS/docker-ce-cli.spec
+++ b/SPECS/docker-ce-cli.spec
@@ -1,8 +1,10 @@
 %global debug_package %{nil}
+%global _origversion 19.03.13
+%global _gitcommit cd8016b6bc
 
 Name: docker-ce-cli
-Version: %{_version}
-Release: %{_release}%{?dist}
+Version: 19.03.13
+Release: 3.el8%{?_tis_dist}.%{tis_patch_ver}
 Epoch: 1
 Summary: The open-source application container engine
 Group: Tools/Docker
-- 
1.8.3.1

